docs/user/spark/spark_jts.rst

```diff
@@ -132,6 +132,38 @@ A complete list of the implemented UDFs is given in the next section (:doc:`./sp
 
 .. _OpenGIS Simple feature access SQL option: http://www.opengeospatial.org/standards/sfs
 
+GeoJSON Output
+^^^^^^^^^^^^^^
+
+The Spark JTS module also provides a means of exporting a ``DataFrame`` to a `GeoJSON <http://geojson.org/>`__ string.
+This allows for quick visualization of the data in many front-end mapping libraries that support GeoJSON input such as
+Leaflet or Open Layers.
+
+To convert a DataFrame, each partition can be map its rows to GeoJSON by instantiating a ``RowGeoJSON`` converter.
+
+.. code-block:: scala
+
+    import org.locationtech.geomesa.spark.jts.util.RowGeoJSON
```

Comment:
thanks for adding the imports - every time i try to execute one of these snippets i have to track down the imports :)